app/femr/ui/views/history/indexPatient.scala.html

```diff
@@ -97,6 +97,8 @@ <h3 class="pull-left bold margin-top-zero">General Info</h3>
                     <input type="text" class="form-control input-sm pull-left" name="firstName" value="@viewModel.getPatientItem.getFirstName" readonly/>
                     <label class="inputLabel">Last Name:</label>
                     <input type="text" class="form-control input-sm pull-left" name="lastName" value="@viewModel.getPatientItem.getLastName" readonly/>
+                    <label class="inputLabel">Phone Number:</label>
+                    <input type="text" class="form-control input-sm pull-left" name="phoneNumber" value="@viewModel.getPatientItem.getPhoneNumber" readonly/>
```

Comment:
1) Excellent job on the pull request :+1: 
2) The way i understand this is that "tel" would limit us to only numbers. "text" would let users input characters. 
3) Do we have a case where a phone number would need a character? 
4) Majority of users are using a tablet